recipes/ctpg/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.3.2":
```

Comment:
1.3.4 is out now, which should support clang

Reply:
Thank you for your information!
I update this PR.